### ISSUE TITLE
(Ubuntu) Rebooting stops all nodes of kubead-dind-cluster

### ISSUE THREAD

**Issue by JiniousChoi** [apache/incubator-openwhisk-deploy-kube#338]
I guess this is because of the default restart policy for docker containers by the docker daemon.
But, simply `docker start <master/node-1/node-2> doesn't bring up the k8s cluster with Openwhisk.

I intend to use kubeadm-dind-cluster for local development purpose and yet I don't see any guide about it on any documents in this project.

For now, I have no other choices but deploy the long-taking openwhisk deployment on cluster all over again, every time. There's gotta be a better way than this, I hope.

**Comment by dgrove-oss** [comment]
I experimented with this a little this afternoon.  I did not find a reliable way to suspend the kubeadm-dind-cluster and later restart it.  I suspect this is a limitation of kubeadm-dind-cluster.  Even ignoring OpenWhisk, I was not able to find a sequence where Helm's `tiller` pod running on the kubeadm-dind-cluster would survive across start/stop of the kubeadm-dind-cluster.   

I documented as a limitation in docs/k8s-didn-cluster.md in #335.
